docs/dev/docker-hub.md

```diff
@@ -26,19 +26,19 @@ such that limiting the number of available cores (with `--smp 1`) is
 the simplest way to go.
 
 Using multiple cores requires setting a proper value to the `/proc/sys/fs/aio-max-nr`.
-On many non-production systems, it will be equal to 65K. The formula
-to calculate the proper value is:
+On many non-production systems, it will be equal to 64K. The formula
```

Comment:
i'd like to take the opportunity to revise this line:

1. use the present tense. as the fact that the number of `aio-max-nr` in many non-production system equals to 64K is a behavior not associated with a particular time.
2. highlight the reason why we put the fact here

so the revised wording can be:

> While the default value for aio-max-nr on many non-production systems is 64K, this may not be optimal for high-performance workloads. The ideal value depends the current value of `/proc/sys/fs/aio-nr` and also on the number of cores to be used by ScyllaD:

Reply:
The original wording ("it will be equal to") struck me too as awkward; however, I considered stylistic updates out of scope for this patch.

That said, I'll adopt your proposal in v2. Thanks!